content/en/methods/page/Markup.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Markup
+description: # TODO
+categories: []
+keywords: []
+action:
+  related: []
+  returnType: # TODO
+  signatures: [PAGE.Markup]
+---
+
+<!-- TODO -->
+
+<!-- JMM It is unclear to me if things like .Page.WordCount will be deprecated in favor of .Page.Markup.CountWords. Three levels (Page, Markup, Countwords) is a bit of a pain to document with the current docs theme. Several of the Markup methods required their own page.
```

Comment:
I will probably make the method names the same (e.g. WordCount and not CountWords) to make it easier for people to migrate (that change was a little bit of a vanity change), but I don't think we want to have the method live in both places forever.